website/www/site/assets/icons/github-icon.svg

```diff
@@ -0,0 +1,10 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" fill="none" viewBox="0 0 32 32">
```

Comment:
You'll need to add a license header on these files, you can find examples in other .svg files in the repo. How were these images created, are they all original (aside from the youtube/twitter/github icons)?

Reply:
Thanks for letting me know. As far as I know, these images are created by our designer.